.github/workflows/ci.yml

```diff
@@ -7,20 +7,32 @@ jobs:
     runs-on: ubuntu-latest
     name: Ubuntu Bionic CI
     steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
       - name: Checkout
         uses: actions/checkout@v2
       - name: Compile and test
         id: ci
-        uses: ignition-tooling/action-ignition-ci@master
+        uses: ignition-tooling/action-ignition-ci@ign-tooling-bionic
```

Comment:
I noticed that this branch is installing Citadel dependencies:

https://github.com/ignition-tooling/action-ignition-ci/compare/master...ign-tooling-bionic

But this PR is targeted at Dome. Maybe we'll need to create separate branches for each distro? I think it would be good to keep using the `master` / `focal` branches if we could, so that all libraries will benefit from the change without us having to open PRs for all of them.

Reply:
I can open a PR in action-ignition-ci to incorporate those changes in the `master` and `focal` branches.

As far as the dependencies, I don't think that the distribution has to exactly match, since all we are doing is installing `apt` packages.  Since we do `apt update` and `install` during the action itself, it should pick up any different dependencies that are necessary.

The point of these containers is to just serve as a cache to speed up a bulk of the apt installs that we have.  They are still flexible to be extended upon.

Reply:
Cool, that sounds good to me :+1:

Reply:
> I think it would be good to keep using the master / focal branches if we could, so that all libraries will benefit from the change without us having to open PRs for all of them.

Come to think of it, we'll need to open PRs to all libraries anyway, to add the `Login to GitHub Container Registry` step, right? Maybe keeping separate `ign-tooling-*` branches makes the transition easier. We just need to keep those in sync with `bionic` and `focal`